department-of-veterans-affairs/abd-vro
# Flyway migration to Create Alembic with Scripts in Support

Adding a flyway migration that creates the alembic user and schema for the CC team. Populating environment variables both locally in docker compose and in LHDI with helm updates. Introducing a create-secret.sh script that provides a generic mechanism for creating kubernetes secrets

## What was the problem?
While some of the work to enable alembic for CC was completed manually, this PR is intended to automate many of the steps that were previously being done manually.

Associated tickets or Slack threads:
- #2886
- https://dsva.slack.com/archives/C01Q7979Z7D/p1713990908740209?thread_ts=1711990256.657119&cid=C01Q7979Z7D

## How does this fix it?[^1]
Automates the manual steps.

## How to test this PR
- Deploy to dev, qa, and sandbox 
- Note: deployment to prod environments is being tracked in #2841


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
